testutil/ensurelogchecker.go

```diff
@@ -124,3 +126,45 @@ func CheckEnsureLoopLogging(filename string, c *check.C, allEnsuresMustContainTr
 		}
 	}
 }
+
+// within a given folder, finds all files that contain an
```

Comment:
this doesn't look like a function doc comment, those need to start with the function name